app/migrations/Version20180529102348.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20180529102348 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        $this->addSql('ALTER TABLE transaction ADD type VARCHAR(255) NOT NULL DEFAULT \'paybox\'');
```

Comment:
`donation_` ?

Reply:
At this point it wasn't renamed ^^, but I will change it